<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginDockWidgetBase</class>
 <widget class="QDockWidget" name="PluginDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoadNetwork</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>391</width>
         <height>759</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="database_connection_name">
               <property name="text">
                <string/>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <layout class="QFormLayout" name="formLayout">
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Plugin</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="plugin_version">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Database</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="database_version">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Project database connection name</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Versions</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="version_comment">
               <property name="text">
                <string/>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Install and configure</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="button_configure_plugin">
             <property name="text">
              <string>Choose database connection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_create_database_structure">
             <property name="text">
              <string>Create database structure</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_create_database_local_interface">
             <property name="text">
              <string>Create database local interface</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_upgrade_database_structure">
             <property name="text">
              <string>Upgrade database structure</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Editing</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="button_import_data">
             <property name="text">
              <string>Import data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_create_editing_session">
             <property name="text">
              <string>Create a new editing session</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Help</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="button_online_help">
             <property name="text">
              <string>Online help</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>button_configure_plugin</tabstop>
  <tabstop>button_create_database_structure</tabstop>
  <tabstop>button_create_database_local_interface</tabstop>
  <tabstop>button_upgrade_database_structure</tabstop>
  <tabstop>button_online_help</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
